src/main/java/core/basesyntax/Drawing.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawing {
+    public abstract void drawFigure();
```

Comment:
better rename method to draw(), without `Figure`

Reply:
done